### PR TITLE
[map] draw map points without data values

### PR DESCRIPTION
**Pull request by chejennifer**
- allow drawing map points even if they don't have associated data values

e.g., map point place type is SuperfundSite, no map point sv, and there is no data on median income for superfund sites
Screenshot 2023-05-30 at 10 00 07 PM

### CODE REVIEW

File: static/js/tools/map/d3_map.tsx [hunk 1]
```diff
@@ -176,23 +176,23 @@ export function D3Map(props: D3MapProps): JSX.Element {
...
       let mapPointSvTitle = "";
       if (statVar.value.mapPointSv !== statVar.value.dcid) {
         mapPointSvTitle =
           statVar.value.mapPointSv in statVar.value.info
             ? statVar.value.info[statVar.value.mapPointSv].title
             : "";
       }
-      const filteredMapPoints = props.mapPoints.filter((point) => {
-        return point.placeDcid in props.mapPointValues;
-      });
```
Comment (pradh) [inline static/js/tools/map/d3_map.tsx:189]: When mapPointValues exist (i.e., mapsv) should we still filter?

**Review by pradh — APPROVED** [review]
Thanks!

File: static/js/tools/map/d3_map.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/map/d3_map.tsx:189]: sure, updated!

**Review by shifucun — CHANGES_REQUESTED** [review]
In general I feel there should be a finer control on what type of points to show here? I don't think we should show all points for unrelated stat var on a map.

File: static/js/tools/map/d3_map.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/map/d3_map.tsx:189]: chatted with Bo about this & we think for consistency, it'll make more sense to show all the points & just show "data unavailable" for points missing data. Also when mapPointValues exist, there aren't going to be a lot of points without data, so it doesn't look too different from before.

Comment (pradh) [reply static/js/tools/map/d3_map.tsx:189]: ok, sg

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Pull request merged**